src/transformers/quantizers/quantizer_torchao.py

```diff
@@ -46,6 +49,20 @@ def find_parent(model, name):
     return parent
 
 
+def _quantization_type(weight):
+    if isinstance(weight, AffineQuantizedTensor):
+        return f"{weight.__class__.__name__}({weight._quantization_type()})"
+
+    if type(weight) is torch.Tensor:
+        return "not quantized"
+
+    return "not recognized"
```

Comment:
Let's only change the representation we have a quantized weight. I'm a bit skeptical with this since we need to check for `AffineQuantizedTensor` and if we add new classes, we will have to update this.

Reply:
Just checked that this is how it is done in `_quantize` and since the function is private, it's better to copy paste and update if needed. Still, I prefer to remove the "not quantized" and "not recognized" cases.